swagger/swagger.yaml

```diff
@@ -0,0 +1,69 @@
+swagger: "2.0"
+info:
+  title: API for Fabric8 Analytics Ingestion
+  version: "0.1"
+basePath: "/api/v1/"
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  /liveness:
+    get:
+      tags: [Service settings]
+      operationId: rest_api.liveness
+      summary: Get job service liveness
+      responses:
+        200:
+          description: Service is alive
+  /readiness:
+    get:
+      tags: [Service settings]
+      operationId: rest_api.readiness
+      summary: Get job service readiness
+      responses:
+        200:
+          description: Service is ready
+  /ingest:
```

Comment:
Can we also add examples here? https://swagger.io/docs/specification/adding-examples/

Reply:
I have raised an issue for this, closing this PR for now.
https://github.com/fabric8-analytics/fabric8-analytics-ingestion/issues/20